code/modules/client/preference_setup/loadout/loadout_utility.dm

```diff
@@ -15,7 +15,12 @@
 
 /datum/gear/utility/codex
 	display_name = "the traveler's guide to vir"
-	path = /obj/item/weapon/book/codex
+	path = /obj/item/weapon/book/codex/lore/vir
+	cost = 0
+
+/datum/gear/utility/corp_regs
+	display_name = "corporate regulations and legal code"
```

Comment:
I don't really want to use their corporate regs so I am likely to disable this on account of being too much of a pain in the ass to update.